orchestrator/cli/domain_gen_helpers/helpers.py

```diff
@@ -0,0 +1,50 @@
+from itertools import groupby
+from typing import Dict, Optional, Set
+
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.orm import Query
+
+
+def sql_compile(sql: Query) -> str:
+    return str(sql.compile(dialect=postgresql.dialect(), compile_kwargs={"literal_binds": True})).replace("\n", "")
+
+
+def generic_mapper(prop_name: str, model_diffs: Dict[str, Dict[str, Set[str]]]) -> Dict[str, Set[str]]:
+    model_by_resource = [
+        (prop_name_value, model_name)
+        for model_name, diff in model_diffs.items()
+        for prop_name_value in diff.get(prop_name, [])
+    ]
+    grouped = groupby(model_by_resource, lambda x: x[0])
+    return {k: {val[1] for val in v} for k, v in grouped}
+
+
+def get_user_input(text: str, default: Optional[str] = None) -> Optional[str]:
+    while True:
+        answer = input(text)
```

Comment:
Might be nice to apply `.strip()` to the user input to get rid of accidental whitespace

Reply:
used it on the returned answer